### PR TITLE
Fixed shared channel doc nconsistencies

### PR DESCRIPTION
**Pull request by devjha-ms**
Fix shared channel doc inconsistencies as pointed out by Angela Chu-Hatoun in https://domoreexp.visualstudio.com/MSTeams/_workitems/edit/2374934/

### CODE REVIEW

**Review by akhilkohlimicrosoft — APPROVED** [review]

File: api-reference/beta/resources/team.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ For a POST request example, see [Request (create team in migration state)](/micr
 |:---------------|:--------|:----------|
 |allChannels|[channel](channel.md) collection|List of channels either hosted in or shared with the team (incoming channels).|
 |channels|[channel](channel.md) collection|The collection of channels and messages associated with the team.|
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/team.md:92]: @devjha-ms , can you add the missing group navigation property to the table as well?

**Review by angelgolfer-ms — CHANGES_REQUESTED** [review]
Hi @devjha-ms , thanks for following up with this PR. Please see my one comment.

File: api-reference/beta/resources/team.md (hunk 1 above)
Comment (devjha-ms) [reply api-reference/beta/resources/team.md:92]: Not added "group" in relationship as we don't expose /teams/{teamId}/group graph API.

Comment (angelgolfer-ms) [reply api-reference/beta/resources/team.md:92]: @devjha-ms , further question to you offline on Teams, please take a look.

**Comment by angelgolfer-ms** [comment]
@devjha-ms , per our Teams chat, I can approve this PR, but we should follow up on the group nav property of the team entity separately.

**Review by angelgolfer-ms — APPROVED** [review]

**Pull request merged**